examples/create_account/main.go

```diff
@@ -48,12 +48,14 @@ func CreateAccountDemo() {
 		SetHashAlgo(crypto.SHA3_256).
 		SetWeight(flow.AccountKeyWeightThreshold)
 
+	referenceBlockId := examples.GetReferenceBlockId(flowClient)
```

Comment:
```suggestion
	referenceBlockID := examples.GetReferenceBlockId(flowClient)
```

Go convention is to always capitalized acronyms/abbreviations: